docker-compose.yml

```diff
@@ -1,53 +1,52 @@
 version: '3.3'
 services:
-    tyk-dashboard:
-        image: tykio/tyk-dashboard:latest
-        ports:
-            - "3000:3000"   
-            - "5000:5000"
-        networks:
-            - tyk
-        depends_on:
-            - tyk-redis
-            - tyk-mongo
-    tyk-gateway:
-        image: tykio/tyk-gateway:latest
-        ports:
-            - "8181:8181"
-        networks:
-            - tyk
-        depends_on:
-            - tyk-redis
-    tyk-pump:
-        image: tykio/tyk-pump-docker-pub:latest
-        networks:
-            - tyk
-        depends_on:
-            - tyk-redis
-            - tyk-mongo
-            - tyk-gateway
-    tyk-redis:
-        image: redis
-        ports:
-            - "6379:6379"
-        volumes:
-            - redis-data:/data
-        networks:
-            - tyk
-    tyk-mongo:
-        image: mongo:3.2
-        command: ["mongod", "--smallfiles"]
-        ports:
-            - "27017:27017"
-        volumes:
-            - mongo-data:/data/db
-        networks:
-            - tyk
+  tyk-dashboard:
+    image: tykio/tyk-dashboard:latest
+    ports:
+    - "3000:3000"
+    - "5000:5000"
+    networks:
+    - tyk
+    depends_on:
+    - tyk-redis
+    - tyk-mongo
+  tyk-gateway:
```

Comment:
Perhaps add a port env var for the gateway docker container anyway? Just to be safe in case something changes there again.